config/routes.rb

```diff
@@ -156,7 +156,8 @@
     patch "/unhide", to: "comments#unhide"
   end
   resources :comment_mutes, only: %i[update]
-  resources :users, only: [:update] do
+  resources :users, only: %i[index], defaults: { format: :json } # internal API
```

Comment:
Rails allows for multiple resource blocks for the same resource, so I'm using this here to apply different defaults for the one that's only part of the internal API.